eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/EventMeshCloudEventWriter.java

```diff
@@ -25,14 +25,14 @@
 
 public class EventMeshCloudEventWriter implements CloudEventContextWriter {
 
-    private Map<String, Object> extensionMap = null;
+    private Map<String, Object> extensionMap;
 
     public EventMeshCloudEventWriter() {
         extensionMap = new HashMap<String, Object>();
     }
 
     @Override
-    public CloudEventContextWriter withContextAttribute(String key, String value)
+    public CloudEventContextWriter withContextAttribute(@Nullable String key, @Nullable String value)
```

Comment:
Missing import:
import javax.annotation.Nullable;

Reply:
thank you for help